esp/src/src-react/components/DFUWorkunits.tsx

```diff
@@ -189,6 +190,25 @@ export const DFUWorkunits: React.FunctionComponent<DFUWorkunitsProps> = ({
             onClick: () => { FileSpray.DFUWorkunitsAction(selection, "SetToFailed"); }
         },
         { key: "divider_3", itemType: ContextualMenuItemType.Divider, onRender: () => <ShortVerticalDivider /> },
+        {
+            key: "restore", text: nlsHPCC.Restore, disabled: !uiState.hasSelection,
+            onClick: () => {
+                if (selection.length === 0) return;
```

Comment:
I think this can be removed, since the button is disabled if there is no selection